test/metabase/lib/join_test.cljc

```diff
@@ -418,3 +418,42 @@
                 (lib/join-condition-operators lib.tu/venues-query lhs rhs)))
         (is (= (lib/join-condition-operators lib.tu/venues-query lhs rhs)
                (lib/join-condition-operators lib.tu/venues-query -1 lhs rhs)))))))
+
+(deftest ^:parallel suggested-join-condition-test
+  (testing "DO suggest a join condition for an FK -> PK relationship"
+    (are [query] (=? {:lib/type :lib/external-op
+                      :operator :=
+                      :args     [{:name "CATEGORY_ID", :id (meta/id :venues :category-id)}
+                                 {:name "ID", :id (meta/id :categories :id)}]}
+                     (lib/suggested-join-condition
+                      query
+                      (meta/table-metadata :categories)))
+      ;; plain query
+      (lib/query meta/metadata-provider (meta/table-metadata :venues))
+
+      ;; query with an aggregation (FK column is not exported, but is still "visible")
+      (-> (lib/query meta/metadata-provider (meta/table-metadata :venues))
+          (lib/aggregate (lib/count))))))
+
+(deftest ^:parallel suggested-join-condition-pk->fk-test
+  (testing "Don't suggest join conditions for a PK -> FK relationship"
```

Comment:
What's the reasoning behind this? Isn't this some arbitrary asymmetry?

Reply:
idk, it's just how it works in MLv1. I was trying to replicate existing behavior so we didn't break a bunch of tests